### PR TITLE
Update intune-devices-userexperienceanalyticsapphealthapplicationperf…

### PR DESCRIPTION
**Pull request by zdvorak**
…ormance.md

Clarification that the allOrgsHealthScore property is not currently in use.

### CODE REVIEW

**Comment by jasonjoh** [comment]
Please disregard the earlier message from msftbot. Your pull request was incorrectly flagged.

**Comment by lramosvea** [comment]
Hi @zhushanms could you please take a look at this PR?

File: api-reference/beta/resources/intune-devices-userexperienceanalyticsapphealthapplicationperformance.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Here is a JSON representation of the resource.
   "appHangCount": 1024,
   "appHealthScore": "4.2",
   "appHealthStatus": "String",
-  "allOrgsHealthScore": "4.2",
+  "allOrgsHealthScore": "0.0",
```
Comment (isvargasmsft) [inline api-reference/beta/resources/intune-devices-userexperienceanalyticsapphealthapplicationperformance.md:63]: Please reproduce this change in any other example when this property is returned, For example Get, List and Update topics.

File: api-reference/beta/resources/intune-devices-userexperienceanalyticsapphealthapplicationperformance.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ The user experience analytics applicaiton performance entity contains app perfor
 |appHangCount|Int32|The number of hangs for the app. Valid values -2147483648 to 2147483647|
 |appHealthScore|Double|The health score of the app. Valid values -1.79769313486232E+308 to 1.79769313486232E+308|
 |appHealthStatus|String|The overall health status of the app.|
-|allOrgsHealthScore|Double|The median health score of the application across all organizations. Valid values -1.79769313486232E+308 to 1.79769313486232E+308|
+|allOrgsHealthScore|Double|This property is not currently in use. If called, this property will always return a value of 0.0|
```
Comment (isvargasmsft) [inline api-reference/beta/resources/intune-devices-userexperienceanalyticsapphealthapplicationperformance.md:36]: PM (Dave Randall) approved this change on the side.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @zdvorak, 

Thank you for bringing this into the docs. I pinged Dave (Intune PM) on the side and he is ok with this change. 

Just 1 additional request: could you please reproduce this update in all the topics where this property is returned? 

Kind regards, 

Isaac Vargas

**Pull request closed without merging**